Pre-release extension installed from marketplace does not work on Ubuntu

**Description**

I installed the vscode-deephaven pre-release extension from Marketplace on Ubuntu and it doesn't seem to start up.

**Steps to reproduce**

1. Install vscode-deephaven extension from Marketplace

**Expected results**

1. Should be able to use the extension, run Deephaven code

**Actual results**

1. The Deephaven extension does not appear in the bottom toolbar. The Run buttons are available on Python files, but don't do anything.

**Additional details and attachments**

Looking in the VSCode Output tab, in the "Extension Host" part of the dropdown there's an exception:

```
2024-07-16 17:00:37.212 [error] Activating extension deephaven.vscode-deephaven failed due to an error:
2024-07-16 17:00:37.212 [error] Error: Cannot find module './Logger'
Require stack:
- /home/bender/.vscode/extensions/deephaven.vscode-deephaven-0.1.3/out/util/index.js
- /home/bender/.vscode/extensions/deephaven.vscode-deephaven-0.1.3/out/extension.js
- /usr/share/code/resources/app/out/vs/loader.js
- /usr/share/code/resources/app/out/bootstrap-amd.js
- /usr/share/code/resources/app/out/bootstrap-fork.js
	at Module._resolveFilename (node:internal/modules/cjs/loader:1055:15)
	at i._resolveFilename (node:electron/js2c/utility_init:2:13405)
	at Module._load (node:internal/modules/cjs/loader:908:27)
	at c._load (node:electron/js2c/node_init:2:13672)
	at S._load (/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:177:6035)
	at n._load (/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:174:31788)
	at i._load (/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:139:34301)
	at Module.require (node:internal/modules/cjs/loader:1122:19)
	at Module.require (/home/bender/.vscode/extensions/github.copilot-chat-0.17.1/dist/extension.js:8:42349)
	at h (/usr/share/code/resources/app/out/vs/loader.js:4:647)
	at Object.<anonymous> (/home/bender/.vscode/extensions/deephaven.vscode-deephaven-0.1.3/out/util/index.js:20:14)
	at o._compile (/usr/share/code/resources/app/out/vs/loader.js:4:1271)
	at Module._extensions..js (node:internal/modules/cjs/loader:1311:10)
	at Module.load (node:internal/modules/cjs/loader:1098:32)
	at Module._load (node:internal/modules/cjs/loader:945:12)
	at c._load (node:electron/js2c/node_init:2:13672)
	at S._load (/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:177:6035)
	at n._load (/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:174:31788)
	at i._load (/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:139:34301)
	at Module.require (node:internal/modules/cjs/loader:1122:19)
	at Module.require (/home/bender/.vscode/extensions/github.copilot-chat-0.17.1/dist/extension.js:8:42349)
	at h (/usr/share/code/resources/app/out/vs/loader.js:4:647)
	at Object.<anonymous> (/home/bender/.vscode/extensions/deephaven.vscode-deephaven-0.1.3/out/extension.js:6:16)
	at o._compile (/usr/share/code/resources/app/out/vs/loader.js:4:1271)
	at Module._extensions..js (node:internal/modules/cjs/loader:1311:10)
	at Module.load (node:internal/modules/cjs/loader:1098:32)
	at Module._load (node:internal/modules/cjs/loader:945:12)
	at c._load (node:electron/js2c/node_init:2:13672)
	at S._load (/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:177:6035)
	at n._load (/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:174:31788)
	at i._load (/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:139:34301)
	at Module.require (node:internal/modules/cjs/loader:1122:19)
	at Module.require (/home/bender/.vscode/extensions/github.copilot-chat-0.17.1/dist/extension.js:8:42349)
	at require (node:internal/modules/helpers:130:18)
	at Function.i [as __$__nodeRequire] (/usr/share/code/resources/app/out/vs/loader.js:5:98)
	at m.xb (/usr/share/code/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:174:32967)
	at async Promise.all (index 0)
```

**Versions**

Extension version: 0.1.3
VSCode version: 1.91.1
OS: Ubuntu 22.04

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "vscode-deephaven",
-  "version": "0.1.3",
+  "version": "0.1.4",
   "displayName": "Deephaven in VS Code",
   "description": "",
   "publisher": "deephaven",
@@ -94,11 +94,13 @@
   },
   "scripts": {
     "vscode:prepublish": "npm run compile",
-    "compile": "tsc -p ./",
+    "clean": "rm -rf out",
+    "compile": "npm run clean && tsc -p ./",
     "watch": "tsc -watch -p ./",
     "pretest": "npm run compile && npm run lint",
     "lint": "eslint src --ext ts",
     "package": "vsce package -o releases/ && vsce package -o releases/vscode-deephaven-latest.vsix",
+    "publish:pre": "vsce publish --pre-release",
     "test": "node ./out/test/runTest.js"
   },
   "devDependencies": {

